editor/editor_export.cpp

```diff
@@ -90,6 +90,19 @@ Ref<EditorExportPlatform> EditorExportPreset::get_platform() const {
 	return platform;
 }
 
+void EditorExportPreset::update_files_to_export() {
+	Vector<Set<String>::Element *> to_remove;
```

Comment:
I think this could be simplified to `Vector<String>` (and `push_back(E->get())`).

Reply:
Made the changes. Thanks!